src/images/rectangle_icon.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>B4E920E2-3014-4C4E-B9C2-A8FD8B23A7F7</title>
```

Comment:
Title here should probably be the alt text for the graphic.

Reply:
Nice! I often forget about the `<aside>`